blog/2022-12-01-governance-changes.md

```diff
@@ -0,0 +1,13 @@
+---
+authors:
+  - name: Antonin Delpeuch
+title: "Governance changes"
+---
+
+Sandra Fauconnier has been OpenRefine's project director since February 2022, and has been working on many fronts in this capacity: leading a reflection on the transformation of the project's governance model, running the biyearly user survey, coordinating our participation in the Outreachy internship programme, migrating from Google Groups to our new Discourse forum, and many more project management tasks behind the scenes. Given her close ties with the Wikimedia and Wikibase communities, she also strengthened the project's ties with those partners.
+
+Sandra has decided to step down from the project director role, returning to her previous work in Linked Open Data in the cultural sector and the Wikimedia movement. She plans to still be involved with OpenRefine's community and will continue to advocate for the tool and to give trainings. As of December 1st, she has transfered back the general project governance responsibilities to the advisory committee, consisting of Martin Magdinier, Jan Ainali and Antonin Delpeuch.
```

Comment:
Please change this to:

Sandra has decided to step down from the project director role, returning to her previous work in Linked Open Data in the cultural sector and the Wikimedia movement. As of December 1st, she has transfered back the general project governance responsibilities to the advisory committee, consisting of Martin Magdinier, Jan Ainali and Antonin Delpeuch.

(I will stay engaged with OpenRefine if we indeed see clear potential in the tool being a viable option for Linked Open Data use cases in the future; this will require a good general assessment first. Can't promise I will keep investing my time in it if we conclude it's not viable or strategically wise for the communities I serve to invest in the tool.)

Also, I feel very uncomfortable with the last sentence's wording. It doesn't sound correct. The project director position is about *implementation* of governance - not about the responsibility for it. That lies, and always has resided, with the project's boards/committees, and with whoever shows up to work on the project (i.e. its contributors), as these people's day to day work, and their interests, are currently the actual force that drives the project's direction.

Reply:
Okay, I have added your version!

> Also, I feel very uncomfortable with the last sentence's wording. It doesn't sound correct. The project director position is about implementation of governance - not about the responsibility for it. That lies, and always has resided, with the project's boards/committees, and with whoever shows up to work on the project (i.e. its contributors), as these people's day to day work, and their interests, are currently the actual force that drives the project's direction.

Do you have a suggestion for a better wording? Happy to include that too.